### ISSUE TITLE
5 Gordon Ln WESTPORT

### ISSUE THREAD

**Issue by HelloMrWillemain** [3DXScape/MobileLocater#48]
Operator entered this ticket, when she mapped the area she happened to be using the "Google" filter. After completing the ticket and submitting, she received and call back from the contractor who told her she marked the wrong area. She looked the ticket up saw the wrong area marked out on the GIF and figured it was her mistake. She cancelled that ticket, created a new one using the same method as before. It happened again. She cancelled the second ticket and created a new one but changed the mapping filter back to the CBYD map. It worked. I have attached multiple screenshots. Why is this an issue of which map someone uses to view? 
![5 gordon ln westport gmap](https://cloud.githubusercontent.com/assets/17441865/13642027/e1b2e386-e5e8-11e5-8747-783dfa7e5bfd.PNG)
![5 gordon ln westport](https://cloud.githubusercontent.com/assets/17441865/13642026/e1b2e692-e5e8-11e5-8bd8-0759f9286bb9.PNG)
![cbyd 5 gordon ln westport gmap filter](https://cloud.githubusercontent.com/assets/17441865/13642025/e1b1dad6-e5e8-11e5-9f65-8350f9831ff3.PNG)

**Comment by twigz13** [comment]
So there is a difference in how the maps displays Gordon Ln and Lazybrook Ln. You can see the difference is the screenshots provided.

I believe that google has the correct display, as the actual location that Polytope returns matches googles map, not its own.

I haven't seen the jobs the customer lodged, but the map displayed on the utility notifications is Polytope. If the contractor was referring to this, and the map was marked according to the google map, the it would look incorrect.

**Comment by 3DXScape** [comment]
I agree with Justin. The locations of the address are correct but one of the maps tiles (probably the Polytope one) is in error. Will investigate and rebuild as appropriate. …steve
